### PR TITLE
Add export form help text, Refs #10009

### PR DESCRIPTION
**Pull request by sbreker**
Added the help icon and expandable help text box to the new CSV/XML export
form.

### CODE REVIEW

**Comment by sevein** [comment]
:+1:

File: js/exportOptions.js [hunk 1]
```diff
@@ -144,6 +146,22 @@
         .append(this.$exportDiv.data('export-alert-message'))
         .prependTo($('#wrapper.container'));
       }
+    },
+
+    toggleGenericHelp: function (e)
+    {
+      e.preventDefault();
+
+      if(this.$genericHelpIcon.toggleClass('open').hasClass('open'))
```
Comment (sevein) [inline js/exportOptions.js]: `if (`

Comment (sbreker) [reply js/exportOptions.js]: Will fix!  :)

File: js/exportOptions.js [hunk 2]
```diff
@@ -144,6 +146,22 @@
...
+      {
+        this.$genericHelpIcon.attr('aria-expanded', true);
+      }
+      else
+      {
+        this.$genericHelpIcon.attr('aria-expanded', false);
+      }
```
Comment (sevein) [inline js/exportOptions.js]: Small suggestion, what do you think?

``` js
var expanded = this.$genericHelpIcon.toggleClass('open').hasClass('open');
this.$genericHelpIcon.attr('aria-expanded', expanded);
```

Comment (sevein) [reply js/exportOptions.js]: (not really important)

**Comment by sevein** [comment]
Sorry I :+1:'ed but on a second thought, should we use the default tooltip description style that we use in our other forms?

**Comment by sevein** [comment]
See `7a3f83bb68a6d29d25f9a56b5a94f4c26a6c2866` (change made in `apps/qubit/modules/settings/templates/pageElementsSuccess.php`). Would that work? It'd make the code much simpler.

**Comment by sbreker** [comment]
Hello! I implemented this to match the help icon on the Adv Search page as requested by Dan.  The implementation of the CSS for the one on the Adv Search page was implemented in a pretty specific way for the dates on that page, so I created a more generic version of the css in the buttons.less file that could potentially be reused.

That commit you reference above just displays the help text as static text in the table cell for me? Is that intended?

**Comment by sevein** [comment]
Yes, that's intended, but I think what you've done is :+1: - please go ahead! I like that you use CSS classes so it's reusable.

File: js/exportOptions.js (hunk 2 above)
Comment (sbreker) [reply js/exportOptions.js]: I like!  Incorporated...

**Comment by sevein** [comment]
Thank you! 👍

**Pull request merged**